### ISSUE TITLE
"name" attribute incorrectly documented for menus

### ISSUE THREAD

**Issue by cmmartti** [rollerworks/NavigationBundle#3]
Either the documentation is incorrect, or the code is buggy. Say you have a very basic menu like this:

    rollerworks_navigation:
        menus:
            mymenu:
                items:
                    -
                        label: My Menu Item
                        route: {name: myroute}

According to the documentation, this is correct. But it doesn't work. It throws an exception and exits, saying:
>The attribute "name" must be set for path "rollerworks_navigation.menus.mymenu.items".

The breadcrumbs work perfectly fine the same way. After a lot of trial and error, it turns out if I change the config like so:

    rollerworks_navigation:
        menus:
            mymenu:
                items:
                    -
                        label: My Menu Item
                        name: my-menu-item
                        route: {name: myroute}

the menu works like it's supposed to, rendering like normal, even though the docs don't say anything about the `name` attribute. Is the right way to do it, or am I missing something else?

Also, it would be great if there were some real-life examples in the documentation. I nearly gave up after a few hours of trying different things. I know how the KnpMenus work and that rendering the menu is as simple as `{{ knp_menu_render('mymenu'}}` in your Twig template, but not everybody knows this, and a tiny push in the right direction like that is a big help sometimes.

**Comment by sstok** [comment]
Thanks for reporting this, it seems the documentation is a little buggy, looking at the tests the following should work:

```yaml
rollerworks_navigation:
    menus:
        mymenu:
            items:
                my-menu-item: # this the actual item name. internally this converted to name rollerworks_navigation.menus.mymenu.items.my-menu-item
                    label: My Menu Item
                    route: {name: myroute}
```

**Comment by sstok** [comment]
Fixed by #5
